typescript/sdk/src/ism/types.ts

```diff
@@ -262,6 +268,10 @@ export const ArbL2ToL1IsmConfigSchema = z.object({
   bridge: z.string(),
 });
 
+export const CCIPReadIsmConfigSchema = z.object({
+  type: z.literal(IsmType.CCIP_READ),
+});
```

Comment:
I would expect the URLs to be included here

Reply:
I also thought so, but it turns out we don't require a view call from `ICcipReadIsm`, so it could technically be dynamic

Reply:
We discussed offline that we can derive this from the message, but will add this later